apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -36,5 +58,251 @@ public void CreateConnection()
             Assert.Equal("my-instance", connection.SpannerInstance);
             Assert.Equal("my-db", connection.Database);
         }
+
+        [Fact]
+        public async Task CreateDatabaseAsync()
+        {
+            // CreateDatabaseAsync will create a new database and dispose of it right after.
+            // But for consistency sake, we want to show the same variables in the snippets.
+            _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+
+            // Sample: CreateDatabaseAsync
+            // Additional: CreateDdlCommand
+            using (var connection =
+                new SpannerConnection($"Data Source=projects/{_projectId}/instances/{_instanceName}/databases/{_databaseName}"))
```

Comment:
Possibly worth having local variables declared outside the snippet so we can use names without underscores?

Reply:
the names are meant to be unresolved when you paste in the snippet.  I don't see the underscore as necessarily bad as we cannot know whether the user will want fields or locals.  It's really just a placeholder for the developer to know he needs to fix (so it generates a compile error on purpose).

Reply:
Yup, I wasn't suggesting trying to make the snippet compile - I just wonder whether the underscore might be a distraction. Not a biggie though.